infra/cifuzz/fuzz_target.py

```diff
@@ -140,6 +145,25 @@ def fuzz(self):
       return FuzzResult(testcase, stderr)
     return FuzzResult(None, None)
 
+  def free_disk_if_needed(self):
+    """Deletes things that are no longer needed from fuzzing this fuzz target to
+    save disk space if needed."""
+    if not self.config.low_disk_space:
+      return
+    logging.info(
+        'Deleting corpus, seed corpus and fuzz target of %s to save disk.',
+        self.target_name)
+
+    # Delete the seed corpus, corpus, and fuzz target.
+    if self.latest_corpus_path and os.path.exists(self.latest_corpus_path):
+      shutil.rmtree(self.latest_corpus_path)
```

Comment:
I'm not sure what's going on but [`rmtree` has failed a couple of times](https://github.com/systemd/systemd/actions/runs/647504148) today with:
```
2021-03-12T22:07:44.0038368Z 2021-03-12 22:07:43,999 - root - INFO - RUN_FUZZERS_MODE is: ci
2021-03-12T22:07:44.0045893Z 2021-03-12 22:07:44,000 - root - INFO - Using OSS-Fuzz as ClusterFuzz deployment.
2021-03-12T22:07:44.0047402Z 2021-03-12 22:07:44,001 - root - INFO - Using address sanitizer.
2021-03-12T22:07:44.0129397Z 2021-03-12 22:07:44,012 - root - INFO - Fuzz targets: ['/github/workspace/out/fuzz-unit-file', '/github/workspace/out/fuzz-varlink']
2021-03-12T22:07:44.0131720Z 2021-03-12 22:07:44,012 - root - INFO - Fuzzer fuzz-unit-file, started.
2021-03-12T22:07:45.3540336Z 2021-03-12 22:07:45,345 - root - INFO - Running command: docker run --rm --privileged --volumes-from 9daab2b8a494 -e OUT=/github/workspace/out -e FUZZING_ENGINE=libfuzzer -e SANITIZER=address -e CIFUZZ=True -e RUN_FUZZER_MODE=interactive gcr.io/oss-fuzz-base/base-runner bash -c run_fuzzer fuzz-unit-file -seed=1337 -len_control=0 -max_total_time=300 /github/workspace/out/cifuzz-corpus/fuzz-unit-file
2021-03-12T22:12:55.3543392Z 2021-03-12 22:12:55,349 - root - ERROR - Fuzzer fuzz-unit-file timed out, ending fuzzing.
2021-03-12T22:12:55.3545473Z 2021-03-12 22:12:55,349 - root - INFO - Deleting corpus, seed corpus and fuzz target of fuzz-unit-file to save disk.
2021-03-12T22:12:55.6500824Z Traceback (most recent call last):
2021-03-12T22:12:55.6543492Z   File "/opt/oss-fuzz/infra/cifuzz/run_fuzzers_entrypoint.py", line 97, in <module>
2021-03-12T22:12:55.6544554Z     sys.exit(main())
2021-03-12T22:12:55.6545760Z   File "/opt/oss-fuzz/infra/cifuzz/run_fuzzers_entrypoint.py", line 83, in main
2021-03-12T22:12:55.6546891Z     result = run_fuzzers.run_fuzzers(config)
2021-03-12T22:12:55.6548405Z   File "/opt/oss-fuzz/infra/cifuzz/run_fuzzers.py", line 208, in run_fuzzers
2021-03-12T22:12:55.6549512Z     if not fuzz_target_runner.run_fuzz_targets():
2021-03-12T22:12:55.6550874Z   File "/opt/oss-fuzz/infra/cifuzz/run_fuzzers.py", line 136, in run_fuzz_targets
2021-03-12T22:12:55.6551928Z     result = self.run_fuzz_target(target)
2021-03-12T22:12:55.6553186Z   File "/opt/oss-fuzz/infra/cifuzz/run_fuzzers.py", line 95, in run_fuzz_target
2021-03-12T22:12:55.6554283Z     fuzz_target_obj.free_disk_if_needed()
2021-03-12T22:12:55.6555619Z   File "/opt/oss-fuzz/infra/cifuzz/fuzz_target.py", line 159, in free_disk_if_needed
2021-03-12T22:12:55.6556854Z     shutil.rmtree(self.latest_corpus_path)
2021-03-12T22:12:55.6557868Z   File "/usr/lib/python3.5/shutil.py", line 478, in rmtree
2021-03-12T22:12:55.6558940Z     onerror(os.rmdir, path, sys.exc_info())
2021-03-12T22:12:55.6559922Z   File "/usr/lib/python3.5/shutil.py", line 476, in rmtree
2021-03-12T22:12:55.6560738Z     os.rmdir(path)
2021-03-12T22:12:55.6562176Z OSError: [Errno 39] Directory not empty: '/github/workspace/out/cifuzz-corpus/fuzz-unit-file'
```

Reply:
I've just opened https://github.com/google/oss-fuzz/issues/5383